src/compass/s1_geocode_metadata.py

```diff
@@ -133,19 +133,26 @@ def run(cfg, burst, fetch_from_scratch=False):
 
     # Dict containing which layers to geocode and their respective file names
     # key: dataset name
-    # value: (bool flag if dataset is to written, raster layer name)
+    # value: (bool flag if dataset is to written, raster layer name, description)
     static_layers = \
-        {file_name_x: (cfg.rdr2geo_params.compute_longitude, 'x'),
-         file_name_y: (cfg.rdr2geo_params.compute_latitude, 'y'),
-         file_name_z: (cfg.rdr2geo_params.compute_height, 'z'),
+        {file_name_x: (cfg.rdr2geo_params.compute_longitude, 'x',
+                       'Longitude coordinate in degrees'),
+         file_name_y: (cfg.rdr2geo_params.compute_latitude, 'y',
+                       'Latitude coordinate in degrees'),
+         file_name_z: (cfg.rdr2geo_params.compute_height, 'z',
+                       'Height in meters'),
          file_name_local_incidence: (cfg.rdr2geo_params.compute_local_incidence_angle,
-                                    'local_incidence'),
+                                    'local_incidence',
+                                     'Local incidence angle in degrees'),
          file_name_los_east: (cfg.rdr2geo_params.compute_ground_to_sat_east,
-                             'los_east'),
+                             'los_east',
+                             'East component of unit vector of LOS from target to sensor'),
          file_name_los_north: (cfg.rdr2geo_params.compute_ground_to_sat_north,
-                              'los_north'),
+                              'los_north',
+                               'North component of unit vector of LOS from target to sensor'),
          file_name_layover: (cfg.rdr2geo_params.compute_layover_shadow_mask,
-                             'layover_shadow_mask')
+                             'layover_shadow_mask',
+                             'Layover shadow mask')
```

Comment:
I think it would be helpful to provide what pixel values are for layover area, shadow area, etc. in the description.